.github/workflows/c-cpp.yml

```diff
@@ -164,8 +121,50 @@ jobs:
       if: runner.os == 'Windows'
       shell: msys2 {0}
       run: |
-       cp /C/_/src/pcem.exe dist
-       zip PCem-Windows-${{ matrix.compiler }}-${{ github.head_ref }}.zip dist/*
+        cp /C/_/src/pcem.exe dist
+        for library in $$($(NTLDD) -R "/C/_/src/pcem.exe" | sed -e 's/^[[:blank:]]*//g' | cut -d ' ' -f 3 | grep -i 'mingw' | sed -e 's|\\|/|g'); do cp "$$library" "dist"; done
```

Comment:
oops, sorry, my snippet was directly from a makefile, which escapes some variables and has the binary name in a variable.

Should probably be something like:
```
for library in $(ntldd -R "/C/_/src/pcem.exe" | sed -e 's/^[[:blank:]]*//g' | cut -d ' ' -f 3 | grep -i 'mingw' | sed -e 's|\\|/|g'); do cp "$library" "dist"; done
```
(Adjust to suit...)